### PR TITLE
ResourceGraph: show warning when too many subscriptions

### PR DESCRIPTION
**Pull request by OlhaTkachenko**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by OlhaTkachenko** [comment]
ToDo: move script to instruction and add link

File: src/ResourceGraph/ResourceGraph/Cmdlets/SearchAzureRmGraph.cs [hunk 1]
```diff
@@ -99,6 +104,16 @@ public override void ExecuteCmdlet()
                 return;
             }
 
+            if (subscriptions.Count > SubscriptionLimit)
```
Comment (cormacpayne) [inline src/ResourceGraph/ResourceGraph/Cmdlets/SearchAzureRmGraph.cs]: @OlhaTkachenko what happens in the response if the user provides more subscriptions than 1000? Is there an exception that's thrown? I just want to make sure there isn't a way for us to call the below script in this cmdlet when the user provides some switch parameter, say `-AllSubscriptions`

**Comment by markcowl** [comment]
@OlhaTkachenko Closing for lack of activity.  Please open a new PR if you would liek these changes merged.

**Pull request closed without merging**